src/main/java/pl/byczazagroda/trackexpensesappbackend/model/User.java

```diff
@@ -67,6 +67,11 @@ public class User implements Serializable {
     @Column(name = "user_status", columnDefinition = "ENUM('VERIFIED', 'UNVERIFIED', 'BLOCKED', 'BANNED')")
     private UserStatus userStatus;
 
+    @OneToMany(mappedBy = "user",
+            cascade = CascadeType.ALL,
+            orphanRemoval = true)
+    private List<FinancialTransactionCategory> financialTransactionCategories = new ArrayList<>();
+
     public void addWallet(Wallet wallet) {
```

Comment:
Please add:
public void addFinancialTransactionCategory(FinancialTransactionCategory financialTransactionCategory){
this.financialTransactionCategories.add(financialTransactionCategory);
financialTransactionCategory.setUser(this);
}

public void removeFinancialTransactionCategory(FinancialTransactionCategory financialTransactionCategory) {
    this.financialTransactionCategories.remove(financialTransactionCategory);
    financialTransactionCategory.setUser(null);
}

Reply:
Done.